### PR TITLE
Documentation update

### PR DESCRIPTION
**Pull request by golaz**
- Updated installation instructions for conda with details for Compy/Cori
- Updated theme layout to increase max pane width to 1200px. Based on
  https://stackoverflow.com/questions/23211695/modifying-content-width-of-the-sphinx-theme-read-the-docs

Addresses  https://github.com/E3SM-Project/e3sm_diags/issues/282

### CODE REVIEW

File: source/install.rst [hunk 1]
```diff
@@ -53,75 +53,105 @@ Note that ``e3sm_unified``'s development cycle is not in phase with ``e3sm_diags
...
+
+If the system doesn't come with conda pre-installed, install it following the
+`official instructions <https://docs.conda.io/projects/conda/en/latest/user-guide/install/>`_. 
+Once installed, it is recommended to upgrade to the latest version:
+
+   ::
+
+       conda update conda
+
+If you are working on a machine/lab that intercepts SSL communications (such as acme1), you will get
```
Comment (forsyth2) [inline source/install.rst]: `machine/network` might be better wording.

File: source/install.rst [hunk 2]
```diff
@@ -53,75 +53,105 @@ Note that ``e3sm_unified``'s development cycle is not in phase with ``e3sm_diags
```
Comment (forsyth2) [inline source/install.rst]: On line 10 can you also change `conda environment pulls together` to `conda environment that pulls together`?

File: source/install.rst [hunk 3]
```diff
@@ -53,75 +53,105 @@ Note that ``e3sm_unified``'s development cycle is not in phase with ``e3sm_diags
...
+If the system doesn't come with conda pre-installed, install it following the
+`official instructions <https://docs.conda.io/projects/conda/en/latest/user-guide/install/>`_. 
+Once installed, it is recommended to upgrade to the latest version:
+
+   ::
+
+       conda update conda
+
+If you are working on a machine/lab that intercepts SSL communications (such as acme1), you will get
+an SSL error unless you disable the check:
```
Comment (forsyth2) [inline source/install.rst]: `disable the SSL verification` might sound better.

File: source/install.rst [hunk 4]
```diff
@@ -53,75 +53,105 @@ Note that ``e3sm_unified``'s development cycle is not in phase with ``e3sm_diags
...
-
-       conda config --set ssl_verify false
-       binstar config --set ssl_verify False
-       conda update conda
+---------------------------
 
+Make sure conda is loaded or installed (see above).
 
-2. Get the developmental yml file to create an environment.
+1. Get the developmental yml file to create an environment.
```
Comment (forsyth2) [inline source/install.rst]: Add `Use ``curl`` if on macOS.` as above

**Comment by golaz** [comment]
@forsyth2 : thanks for your careful proofreading. I made the changes you suggested.

File: source/install.rst [hunk 5]
```diff
@@ -86,8 +86,8 @@ Once installed, it is recommended to upgrade to the latest version:
 
        conda update conda
 
-If you are working on a machine/lab that intercepts SSL communications (such as acme1), you will get
-an SSL error unless you disable the check:
+If you are working on a machine/network that intercepts SSL communications (such as acme1), you will get
+an SSL error unless you disable the SSL verification
```
Comment (forsyth2) [inline source/install.rst]: Add `:`

**Pull request merged**